zebra/zebra_mlag.c

```diff
@@ -88,8 +647,542 @@ void zebra_mlag_init(void)
 {
 	install_element(VIEW_NODE, &show_mlag_cmd);
 	install_element(ENABLE_NODE, &test_mlag_cmd);
+
+	/*
+	 * Intialiaze teh MLAG Global variableis
```

Comment:
"the" , "variables" , "created" please
is "MCLAG" correct - is that "MLAGD" or "MCLAGD" for instance?

Reply:
Taken care, different vendors are using different Nomenclature, will use MCLAG all the places